query/aql_processor.go

```diff
@@ -1346,6 +1364,22 @@ func copyHostToDevice(vps memCom.HostVectorPartySlice, deviceVPSlice deviceVecto
 }
 
 func hostToDeviceColumn(hostColumn memCom.HostVectorPartySlice, device int) deviceVectorPartySlice {
+	if memCom.IsArrayType(hostColumn.ValueType) {
+		deviceColumn := deviceVectorPartySlice{
+			length:       hostColumn.Length,
+			valueType:    hostColumn.ValueType,
+			defaultValue: hostColumn.DefaultValue,
+		}
+		totalColumnBytes := hostColumn.ValueBytes + hostColumn.Length*8
```

Comment:
comment with offset vector bytes for the length * 8 part or just add a variable called OffsetBytes